cartofacade/carto_facade_test.go

```diff
@@ -227,8 +232,7 @@ func TestStop(t *testing.T) {
 		return nil
 	}
 	cartoFacade.carto = &carto
-	cartoFacade.Initialize(cancelCtx, 5*time.Second, &activeBackgroundWorkers)
-	defer cartoFacade.Terminate(cancelCtx, 5*time.Second)
+	cartoFacade.start(cancelCtx, &activeBackgroundWorkers)
```

Comment:
This only needed `cartoFacade.start` to be called, which happened to be called by `Initialize` even though it was returning an unchecked error. 

B/c this test doesn't run any C code we don't need to run Initialize & Terminate. This makes this removes misleading setup logic from the original implementation.